src/app/utils/counterItems.js

```diff
@@ -0,0 +1,8 @@
+export default function counterItems(array) {
+  const counter = array.length;
+  const counterPok = document.querySelector('.counterItems');
+  const counterNav = document.querySelector('.counterPokemons');
+  counterPok.innerHTML = `Choose between our ${counter} pokemons`;
+  counterNav.innerHTML = `(${counter})`;
+  return counter;
+}
```

Comment:
- [ ] Kindly use `Es6` syntax for creating a function because it was specified in the project that you should use `Es6`. It would look something like this:

`export default const functionName = (parameters) => {}`